report_pre_thesis/src/sections/related_work.tex

```diff
@@ -27,17 +26,17 @@ \section{Related work}\label{sec:related_works}
 
 \citeauthor{castorena_deep_2021} developed a deep spectral convolutional neural network (CNN) for processing and analyzing laser-induced breakdown spectroscopy (LIBS) signals.
 Their CNN is designed to first remove uncertainties from sensor data and then assess the chemical content qualitatively and quantitatively from a sample's LIBS signal.
 Their experiments showed that their method surpasses existing ones used by the Mars Science Laboratory for pre-processing and calibration.
 Notably, their CNN operates in real-time and does not require additional information like dark current, system response, temperature, or range.
 For chemical content estimation, their CNN matches current techniques' performance and even exceeds them when utilizing an end-to-end learning approach, all the while reducing prior information requirements and being ready for immediate deployment\cite{castorena_deep_2021}.
-
 Their development of a deep spectral CNN for analyzing LIBS signals showcases an innovative method that excels in real-time operation and minimal data dependency. Their method's ability to operate in real-time and with reduced data dependency provides a compelling framework for future considerations in our work, especially when evaluating the balance between model complexity and operational efficiency in reducing overall error.
 
-\citeauthor{chen_xgboost_2016} introduced XGBoost, a scalable end-to-end tree boosting system that has gained widespread use among data scientists for its state-of-the-art performance in various machine learning challenges. The system is particularly notable for its novel sparsity-aware algorithm tailored for sparse data and the introduction of a weighted quantile sketch method for approximate tree learning. These advancements allow XGBoost to efficiently handle large-scale data with significantly reduced resource requirements. Utilizing cache access patterns, data compression, and sharding, the authors were able to significantly enhance the scalability and efficiency of XGBoost. These approaches enabled it to process a large number of examples with fewer resources compared to other exisiting systems\cite{chen_xgboost_2016}.
-
-While XGBoost's emphasis on scalability and computational efficiency diverges from our current work on enhancing the MOC model, the methodologies it employs could be instrumental for future developments in our research. The efficient handling of large datasets, a key feature of XGBoost, provides a framework that could be adapted to improve our model's performance with extensive data.
+In their work, \citeauthor{chen_xgboost_2016} presented the development of XGBoost, an innovative tree boosting system that has become popular among data scientists for its performance in diverse machine learning applications. 
```

Comment:
Wondering if it makes more sense to talk about XGBoost's effectiveness in creating accurate models, rather than it's efficiency at handling large datasets. Particularly when our datasets aren't very large, as evident by the many papers complaining about the lack of data.

Reply:
Perhaps write something to the effect of:
```
These features enable XGBoost to handle data efficiently and create accurate models, while requiring fewer resources than traditional systems.
```

Reply:
Use some of the information from the paper?

![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/5521bc52-9d54-4851-a371-154e0c75d4e6)

Reply:
How about this then:
```
XGBoost's robust performance, evidenced by its predominant use in winning solutions of various Kaggle challenges, highlights its effectiveness in creating accurate models, even when dealing with smaller datasets.
```